packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/04-upgradeable-plugin/05-updating-versions.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Upgrading an Upgradeable Plugin
+---
+
+## How to upgrade an Upgradeable Plugin
+
+Updating an Upgradeable plugin means we want to direct the implementation of our functionality to a new build, rather than the existing one.
+
+In this tutorial, we will go through how to update an Upgradeable plugin and each component needed.
+
+### 1. Create the new build implementation contract
```

Comment:
We are explaining how to do another build, but the chapter before was already showing builds 2 and 3. Should we simplify the previous chapter?

Reply:
As far as I understand it, the "Subsequent builds" section is about upgrading the Upgradeable contract implementation, whereas the "Updating versions" section is about updating the plugin to a new version.

Key difference is storage is not lost during upgrading, and it is lost when updating.

Potentially @Michael-A-Heuer can provide some aditional guidance here.

Reply:
Yes, we cannot simplify them. Just leave it as is